plugins/available/alias-completion.plugin.bash

```diff
@@ -52,7 +52,7 @@ function alias_completion {
 		read -a alias_arg_words <<< "$alias_args"
 
 		# skip alias if there is no completion function triggered by the aliased command
-		if [[ ! " ${completions[*]} " =~ $alias_cmd ]]; then
+		if [[ ! $(_bash-it-array-contains-element "$alias_cmd" "${completions[@]}") ]]; then
```

Comment:
Now that we're not using `=~` ( `==`), I don't think we need the `[[ $() ]]` wrapping.  Ie seems like this should work:
```
if ! _bash-it-array-contains-element "$alias_cmd" "${completions[@]}"; then
```

Reply:
nice one, done!